makefile

```diff
@@ -0,0 +1,12 @@
+go.fmt:
```

Comment:
Nit-pick: makefiles are usually named with a capital M. Is there a reason not to?